testfiles/statements.titan

```diff
@@ -14,5 +14,9 @@ local function statements(): nil
         print("Hello", "World")
     end
 
+    for i = 1, 2, 3 do
```

Comment:
I can't get my mind to accept that this loop iterates only 1 instead of 1 2 3 :)